packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -129,14 +115,26 @@ public function process(
 
         $timeoutInSeconds = $this->parameterProvider->provideIntParameter(Option::PARALLEL_JOB_TIMEOUT_IN_SECONDS);
 
-        for ($i = 0; $i < $numberOfProcesses; ++$i) {
-            // nothing else to process, stop now
-            if ($jobs === []) {
-                break;
-            }
+        $workerCommandLineFactory = $this->workerCommandLineFactory;
```

Comment:
why making it a variable?

Reply:
good catch, in the first iteration I was not sure yet how $this did behave in anonymous functions, so I avoided it. Need to clean this one up